.github/workflows/tests.yml

```diff
@@ -258,24 +245,22 @@ jobs:
 
     steps:
       - name: Check out repository code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           submodules: true
-      - name: Install packages
-        run: sudo apt-get install xvfb libgbm1
```

Comment:
Does anyone know why these aren't needed anymore? (same for `sudo` above)

Reply:
For `sudo` and `xvfb`, they are included in the ubuntu image, see [here](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#installed-apt-packages) for a list of "all" installed apt packages. While `libgdm1` is not listed specifically, pupeeter works with the  `libgdm-dev`, which is listed. If I remember, correctly, `libgdm1` was actually installed in the image, as I removed that line